Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM	rust:1 AS builder
+
+COPY	. /opt/sekursranko/
+
+RUN	cd /opt/sekursranko \
+&&	cargo build --release
+
+FROM	debian:stretch-slim
```

Comment:
I would prefer an `alpine:3.8` based image, to reduce image size and attack surface.

For this to work, we have to build the Rust binary against the musl target:

- When building, before the `COPY` instruction, add `RUN rustup target add x86_64-unknown-linux-musl`
- In the `cargo build` command, add `--target x86_64-unknown-linux-musl`
- When copying the binary from the builder image, the directory is now `target/x86_64-unknown-linux-musl/release/`

Since this is just a static binary, we don't need any system dependencies, so Alpine is more than enough for our purposes :slightly_smiling_face:

Reply:
> For this to work, we have to build the Rust binary against the musl target:

I tried using alpine but had issues with libgcc / libc-6. Thanks for your pointers, I will take a look.

Reply:
Yes, that happens when building the binary dynamically on a GNU based system like Debian. Alpine uses musl libc instead of glibc, so `libgcc` is not available :slightly_smiling_face:

Reply:
> Alpine uses musl libc instead of glibc, so `libgcc` is not available

Yes I know what Alpine is. But I was to lazy to find out how I would properly cross compile and I prefer the Debian images myself anyway :smile: